### PR TITLE
Added mention of pure ML-DSA and text about CRLs.

### PR DESCRIPTION
**Pull request by csosto-pk**
Added text to mention pure ML-DSA.

Also added text while pure ML-DSA is fine for X.509 and CRLs.

@bwesterb , @seanturner please check.

### CODE REVIEW

File: draft-ietf-lamps-dilithium-certificates.xml [hunk 1]
```diff
@@ -119,6 +119,12 @@
 	      <xref target="RFC3279" format="default"></xref> did for classic cryptography and
 	      <xref target="RFC5480" format="default"></xref> did for elliptic curve cryptography.
 	      The private key format is also specified.</t>
+      <t><xref target="FIPS204" format="default"></xref> defines two versions of ML-DSA, the pure and the 
+	      pre-digest version. Only the former is specified for use in this document. 
+	      The size of the X.509 content signed in most certificates is small enought to not pose a
```
Comment (jakemas) [inline draft-ietf-lamps-dilithium-certificates.xml:124]: typo "enough"

File: draft-ietf-lamps-dilithium-certificates.xml [hunk 2]
```diff
@@ -119,6 +119,12 @@
 	      <xref target="RFC3279" format="default"></xref> did for classic cryptography and
 	      <xref target="RFC5480" format="default"></xref> did for elliptic curve cryptography.
 	      The private key format is also specified.</t>
+      <t><xref target="FIPS204" format="default"></xref> defines two versions of ML-DSA, the pure and the 
+	      pre-digest version. Only the former is specified for use in this document.
```
Comment (bwesterb) [inline draft-ietf-lamps-dilithium-certificates.xml:123]: I propose we remove "The size [...]  ML-DSA performance.", and add

> Pure ML-DSA (in contrast to SLH-DSA) allows for streaming the message that has to be signed. The only advantage of the pre-digest version, is that it allows a different hash to be used for hashing the message.

Comment (csosto-pk) [reply draft-ietf-lamps-dilithium-certificates.xml:123]: The pre-digest people do not argue that you cannot stream the message. Their point is that maybe I don't want to stream it to the HSM over its slow bus. I may want to just send the digest of the message and sign that especially if the message will be signed a lot of times.

**Comment by seanturner** [comment]
Where did we end up on this one? Are we going to add some tweaked text?

**Comment by csosto-pk** [comment]
Give me a little time. 

I think we will need HashML-DSA support in our draft although I was against it all along. I recently found out that some HSMs are very much against large MB or even size messages. 

I am trying to confirm with other vendors to make sure this is not unique to the things I am hearing here and there we will have to tweak the text.

File: draft-ietf-lamps-dilithium-certificates.xml (hunk 1 above)
Comment (seanturner) [inline draft-ietf-lamps-dilithium-certificates.xml:124]:
```suggestion
	      The size of the X.509 content signed in most certificates is small enough to not pose a 
```

**Pull request closed without merging**